app/views/ukrentaroom/allowances/RaRAllowancesCompleteView.scala.html

```diff
@@ -0,0 +1,47 @@
+@*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import controllers.ukrentaroom.allowances.routes._
+
+@this(
+  layout: templates.Layout,
+  formHelper: FormWithCSRF,
+  govukErrorSummary: GovukErrorSummary,
+  govukRadios: GovukRadios,
+  govukButton: GovukButton
+)
+
+@(form: Form[_], taxYear: Int, mode: Mode)(implicit request: Request[_], messages: Messages)
+
+@layout(pageTitle = title(form, messages("ukrentaroom.allowancesComplete.title"))) {
+  @formHelper(action = RaRAllowancesCompleteController.onSubmit(taxYear), Symbol("autoComplete") -> "off") {
+
+  @if(form.errors.nonEmpty) {
+   @govukErrorSummary(ErrorSummaryViewModel(form))
+  }
+
+
+@govukRadios(
+   RadiosViewModel.yesNo(
+    field = form("rarAllowancesCompleteYesOrNo"),
+    legend = LegendViewModel(messages("ukrentaroom.allowancesComplete.heading")).asPageHeading()
```

Comment:
I have discussed this with @arunkumar461 and I thins the font size in legend should be large - `.asPageHeading(LegendSize.Large)`

Reply:
Incorporated. Thanks